src/components/App.tsx

```diff
@@ -23,11 +23,17 @@ function App() {
 			console.log('Handling teleop data:', teleop);
 			// You can add more logic to process the teleop data
 		}
+		function teamData(teamData: IMatch): void {
```

Comment:
Function names should typically start with a verb to describe what it does - in this case, we should rename it to something like `submitMatchData`